vertex/extract_metrics.py

```diff
@@ -29,9 +30,14 @@
     '--mask_file', help='path to CIVET midline mask',required=True)
 group.add_argument(
     '--output_suffix', help='suffix to add to output file name before extension, e.g. _smoothed', type=str, default="")
+group.add_argument(
+    '--save_mat', help='option to save matrix as .mat, otherwise saves as .npz', required=False, action='store_true')
```

Comment:
Default must remain the same as after adding this code, so it should default to save_mat.

Change option to --save_npz

Reply:
done